tests/numpy/numeric_test.py

```diff
@@ -368,6 +371,19 @@ def test_abs(self, num_type):
         with pytest.raises(TypeError):
             ak.abs(np.array([range(0, 10)]).astype(num_type))
 
```

Comment:
That really feels like a disagreement over programming style, since it doesn't change the functionality.  But I'll make the change.

Reply:
It has to be getattr(ak,op) and getattr(np,op) because they've been imported via their abbreviated names.  But the change is in there, and passes tests.  I have some more local testing to do, and then will push it up.

Reply:
Thanks!

The result is actually slightly differently in terms of how pytest runs the test.  If you run them from pycharm you can see that when a parameter is parametized, it splits them into separate tests, each named with the parameter value.  Then, when one fails, it is obvious which value is the culprit.  So it does make debugging a bit easier.